.ci/install_kata_image.sh

```diff
@@ -50,7 +50,7 @@ build_rust_image() {
 			fi
 			distro="${osbuilder_distro:-ubuntu}"
 			if [ ${CCV0} == "yes" ]; then
-				sudo -E USE_DOCKER="${use_docker:-}" DISTRO="${distro}" SKOPEO_UMOCI=yes make -e "image"
+				sudo -E USE_DOCKER="${use_docker:-}" DISTRO="${distro}" SKOPEO="" UMOCI=yes make -e "image"
```

Comment:
2. Can't we remove `SKOPEO=""` altogether?

Reply:
Good point. Removed.